a1/src/main/java/ro/utcn/spet/a1/A1Application.java

```diff
@@ -0,0 +1,13 @@
+package ro.utcn.spet.a1;
```

Comment:
It is very good that you used my examples, but it might be an idea to rename the top level package:P